wiki/Tournaments/SOLMT/2019/en.md

```diff
@@ -0,0 +1,341 @@
+---
+tags:
+- SOLMT
+- SOLMT
+---
+
+# Speed of Light Mania Tournament
+
+![SOLMT logo](img/logo.png)
+
+The **Speed of Light Mania Tournament** (***SOLMT***) is a head to head (1vs1) based Mania tournament. It is the 1st installment of the Speed of Light Mania Tournament.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase |2019-10-14 / 2019-11-6|
+| Qualifiers |2019-11-23 / 2019-11-24|
+| Group Stage First Week|2019-11-30 / 2019-12-01|
+| Group Stage Second Week|2019-12-07 / 2019-12-08|
+| Round of 32 |2019-12-14 / 2019-12-15|
+| Round of 16 |2019-12-21 / 2019-12-22|
+| Quarterfinals |2019-12-27 / 2019-12-28|
+| Semifinals |2020-01-4 / 2020-01-5|
+| Finals |2020-01-11 / 2020-01-12|
+| Grand Finals |2020-01-18 / 2020-01-19|
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for Speed of Light winners. It is possible that these items change with every installment of the tournament.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") |Razer Huntsman Elite (Red Switch) + Tournament Badge + 3 Months of Support |
```

Comment:
```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Razer Huntsman Elite (Red Switch), Tournament Badge, 3 Months of osu!supporter tag |
```
Same, spacing between the content.
Instead using `+ (plus)`,  use `, (comma)`.
Apply to prizes below too.